shared/src/main/kotlin/com/egm/stellio/shared/web/CustomWebFilter.kt

```diff
@@ -0,0 +1,65 @@
+package com.egm.stellio.shared.web
+
+import com.egm.stellio.shared.util.JSON_LD_MEDIA_TYPE
+import com.egm.stellio.shared.util.JSON_MERGE_PATCH_MEDIA_TYPE
+import com.egm.stellio.shared.util.isAcceptable
+import org.springframework.http.HttpMethod
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.http.server.reactive.ServerHttpRequest
+import org.springframework.stereotype.Component
+import org.springframework.web.server.ResponseStatusException
+import org.springframework.web.server.ServerWebExchange
+import org.springframework.web.server.WebFilter
+import org.springframework.web.server.WebFilterChain
+import reactor.core.publisher.Mono
+
+@Component
+class CustomWebFilter : WebFilter {
+
+    override fun filter(exchange: ServerWebExchange, chain: WebFilterChain): Mono<Void> {
+        val request: ServerHttpRequest = exchange.request
+        return when (request.method) {
```

Comment:
Did you check if we still need all of them?

Reply:
Do we?

Reply:
I removed the check for the content-type.
We are checking:
- accept header for get requests
- content length header for post/patch requests
Yes we still need this

Reply:
I'm very surprised that content negociation can't be done by the framework